README.md

```diff
@@ -0,0 +1,59 @@
+# creditrisk-poc
+
+Creditrisk-poc is a Hydra powered API which serves loan portfolio data using EBA NPL Template.
+
+## Features
+creditrisk-poc consist following features:
+* Loan & Borrower classes.
+* Borrower class collection.
+* Loan & Borrower class are linked using a foreign key ("CounterpartyId").
+* Loan class can perform all the CRUD operations ( GET, PUT, POST, DELETE).
+* Borrower can perform all the CRUD operations.
+* Borrower class collection can perform all the CRUD operations.
+
+## API_DOC
+API_Doc is generated through hydra-python-core module doc_writer.
+
+API_Doc & doc_writer file can be found here :
+```
+API_Doc
+|
+|___ APIDOC
+|___ Creditrisk_api_docwriter.py
+```
+`APIDOC` is a pickle serialized object, It can be accessed as follows:
+```python
+import pickle
+
+API_Doc_file = open("creditrisk_poc/API_Doc/APIDOC","rb")
+doc = pickle.load(API_Doc_file)
```

Comment:
@Purvanshsingh why is this being merged? it is using the `pickle` serialisation that we agreed to change.
#3 should have been merged first and this one only after, please be more attentive

Reply:
@chrizandr this should have not been approved

Reply:
@Mec-iS  I'll fix that now, it was a small change else everything was approved by phillipos that's why I merged.

Reply:
We had approved this on the call. I thought it was already reviewed by Phillipos as well.